src/languages/lib/css-shared.js

```diff
@@ -153,7 +156,7 @@ export const MEDIA_FEATURES = [
   'max-width',
   'min-height',
   'max-height'
-];
+].sort().reverse();
```

Comment:
SHouldn't our lists already be sorted, or is this just a failsafe?

Reply:
It's a failsafe: ideally this should be a custom lint or something, but this works for now.